httpdocs/js/webots-cloud.js

```diff
@@ -187,6 +197,9 @@ document.addEventListener('DOMContentLoaded', function() {
     function simulationRow(data) {
       const admin = project.email ? project.email.endsWith('@cyberbotics.com') : false;
       const words = data.url.substring(19).split('/');
+      const dotIdx = data.url.lastIndexOf('/') + 1;
+      const thumbnailUrl = (data.url.slice(0, dotIdx) + "." + data.url.slice(dotIdx)).replace('github.com', 'raw.githubusercontent.com').replace('/blob', '').replace('.wbt', '.jpg');
```

Comment:
Coding style: [no abbreviations](https://github.com/cyberbotics/webots/wiki/Coding-Style/#cs44-no-abbreviations)
```suggestion
      const dotIndex = data.url.lastIndexOf('/') + 1;
      const thumbnailUrl = (data.url.slice(0, dotIndex) + "." + data.url.slice(dotIndex)).replace('github.com', 'raw.githubusercontent.com').replace('/blob', '').replace('.wbt', '.jpg');
```